tests/schemas.test.ts

```diff
@@ -51,6 +51,387 @@ describe('schemas', () => {
           version: 'zora-01012021'
         }
 
+        const result = validator.validate(json)
+        expect(result).toBe(true)
+      })
+    })
+  })
+  describe('catalog', () => {
+    describe('20210202', () => {
+      it('requires all keys', () => {
+        const validator = new Validator('catalog-20210202')
+        const json = {
+          body: {
+            version: "catalog-20210202",
+            title: "Never Gonna Give You Up",
+            notes: "An unexpected classic"
+          }
+        }
+
+        const result = validator.validate(json)
+        expect(result).toBe(false)
+      })
+
+      it('does not allow additional properties', () => {
+        const validator = new Validator('catalog-20210202')
+        const json = {
+          body: {
+            version: "catalog-20210202",
+            title: "Never Gonna Give You Up",
+            artist: "Rick Astley",
+            notes: "An unexpected classic",
+            duration: 213.1,
+            grammyNominated: "I wish",
+            mimeType: "audio/aiff",
+            trackNumber: null,
+            project: null,
+            artwork: {
+              isNft: false,
+              info: {
+                uri: "https://img.discogs.com/Q8B0-mvrVFLPx5jZLlRa2zialII=/fit-in/584x579/filters:strip_icc():format(jpeg):mode_rgb():quality(90)/discogs-images/R-995860-1300387504.jpeg.jpg",
+                mimeType: "image/jpeg"
+              },
+              nft: null
+            },
+            visualizer: {
+              isNft: false,
+              info: {
+                uri: "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
+                mimeType: "video/mp4"
+              },
+              nft: null
+            }
+          },
+          signature: {
+            algorithm: "secp256k1",
+            encoding: "rlp",
+            publicKey: "0xc236541380fc0C2C05c2F2c6c52a21ED57c37952",
+            signature: "0x2d7c0dc8a9252bb8cf0e654c58badb0585f41941270765e46c238a1692243e6d128bbaf072c2886348d49498794365f60f1793a758a7d1c281affc9c81de61ae1b"
+          }
+        }
+
+        const result = validator.validate(json)
+        expect(result).toBe(false)
+      })
+      it('does not allow forbidden audio file formats', () => {
+        const validator = new Validator('catalog-20210202')
+        const json = {
+           body: {
+            version: "catalog-20210202",
+            title: "Never Gonna Give You Up",
+            artist: "Rick Astley",
+            notes: "An unexpected classic",
+            duration: 213.1,
+            grammyNominated: "I wish",
```

Comment:
`nit`: i think `grammyNominated` is an `additionalProperty` mind removing it and confirming the validation fails.